stablehlo/reference/Element.cpp

```diff
@@ -350,6 +350,33 @@ Element min(const Element &e1, const Element &e2) {
       });
 }
 
+// These cases are implementation-defined, and the behavior of this interpreter:
+// For integers: if rhs is 0, the result is the same as `lhs`.
+//   For signed integers, if lhs is INT_MAX and rhs is -1, the result is 0.
+// For floats: use APFloat.mod(). If rhs is 0, the result is 0x7FF8000000000000
+Element rem(const Element &e1, const Element &e2) {
+  return map(
+      e1, e2,
+      [&](APInt lhs, APInt rhs) {
+        if (rhs == 0) return lhs;
```

Comment:
Why do we need this check at all?

Reply:
Without this, there will be an crash with error message from
```
Evaluated results of function: remainder_op_test_si64
stablehlo-interpreter: .../stablehlo/llvm-project/llvm/lib/Support/APInt.cpp:1667:
llvm::APInt llvm::APInt::urem(const llvm::APInt&) const: 
Assertion `RHS.U.VAL != 0 && "Remainder by zero?"' failed.
PLEASE submit a bug report to https://github.com/llvm/llvm-project/issues/ and include the crash backtrace.
Stack dump:
...
```

I thought this was unacceptable. But after rethink, this seems good enough.
Removed that line of code. Please "resolve this comment" if it looks good.